### PR TITLE
Many Flot redraw when someone resize window.

### PR DESCRIPTION
**Pull request by devginie**
When someone resize browser window, every flot directive will be redrawed.

### CODE REVIEW

File: angular-flot.js [hunk 1]
```diff
@@ -151,6 +156,9 @@ angular.module('angular-flot', []).directive('flot', ['$timeout', function ($tim
       //
 
       element.on('$destroy', function onDestroy () {
+
```
Comment (lvillani) [inline angular-flot.js]: I'd remove this blank line.

File: angular-flot.js [hunk 2]
```diff
@@ -151,6 +156,9 @@ angular.module('angular-flot', []).directive('flot', ['$timeout', function ($tim
       //
 
       element.on('$destroy', function onDestroy () {
+        
+        $(window).off('resize'); // remove the handler added earlier
```
Comment (lvillani) [inline angular-flot.js]: This comment is unnecessary.

**Comment by lvillani** [comment]
Looks good but you should address the comments I added to the patch by rewriting the commit (i.e.: do not push a new commit making changes I suggested). I'll merge this PR once that is done.

File: angular-flot.js (hunk 2 above)
Comment (mebibou) [inline angular-flot.js]: I think the worst is not the comment but the fact that this would unbind all `resize` handlers on the window

**Comment by mebibou** [comment]
Actually I don't think it looks so good: there is a [resize](http://www.flotcharts.org/flot/examples/resize/) plugin already made for that, not sure why implementing it here. Also, if it would be done here, it shouldn't be done on every tick, it should probably be done every 200ms or something otherwise performances are gonna be pretty bad on resize.

**Comment by lvillani** [comment]
Ah, right @mebibou, thanks for catching that.

@devginie I'm closing this PR and I, too, suggest you to use the resize plugin.

**Comment by devginie** [comment]
Sorry, I am short on ideas.
This problem is already discussed at #8.

**Pull request closed without merging**